### PR TITLE
updating scripts

### PR DESCRIPTION
**Pull request by ddiaz006**
• Added the control regions to the checker, submitjobs, hadd scripts
• Cleaned up my comments from last PR
• Tested with ntuples in addpuAOD

### CODE REVIEW

**Comment by benjaminkreis** [comment]
@ddiaz006 can you update the .sh scripts such as makelists.sh and submitjobs.sh to all be consistent with what we need to make the stack plots you've been making?

**Comment by benjaminkreis** [comment]
testing now

File: plotters/plotter_stacked.C [hunk 1]
```diff
@@ -542,69 +567,78 @@ void plotter_stacked()
...
     h_VV = (TH1F*)h_WWToLNuLNu->Clone("h_VV");
+     h_VV->Add(h_WW             ) ;
+//     h_VV->Add(h_WG             ) ;
+     h_VV->Add(h_WWToLNuLNu     ) ;
      h_VV->Add(h_WWToLNuQQ      ) ;
      h_VV->Add(h_WZToL3Nu       ) ;
      h_VV->Add(h_WZTo3LNu       ) ;
      h_VV->Add(h_WZToLNu2QorQQ2L) ;
+//     h_VV->Add(h_ZG             ) ;
+     h_VV->Add(h_ZZ             ) ;
```
Comment (benjaminkreis) [inline plotters/plotter_stacked.C:600]: double counting

File: plotters/plotter_stacked.C [hunk 2]
```diff
@@ -542,69 +567,78 @@ void plotter_stacked()
...
      h_ST->Add(h_STbar_tW);
      h_ST->Add(h_ST_tW);
 
-//    h_ZH = (TH1F*)h_ZH_HToBB_ZToLL->Clone("h_ZH");
-//     h_ZH->Add(h_ggZH_HToBB_ZToLL);
+    h_ZH = (TH1F*)h_ZH_HToBB_ZToLL->Clone("h_ZH");
+     h_ZH->Add(h_ggZH_HToBB_ZToLL);
 
     h_VV = (TH1F*)h_WWToLNuLNu->Clone("h_VV");
+     h_VV->Add(h_WW             ) ;
```
Comment (benjaminkreis) [inline plotters/plotter_stacked.C:592]: double counting

File: plotters/plotter_stacked.C [hunk 3]
```diff
@@ -542,69 +567,78 @@ void plotter_stacked()
...
-//    h_GJets = (TH1F*)h_GJets_HT40To100->Clone("h_GJets");
-//     h_GJets->Add(h_GJets_HT100To200);
-//     h_GJets->Add(h_GJets_HT200To400);
-//     h_GJets->Add(h_GJets_HT400To600);
-//     h_GJets->Add(h_GJets_HT600ToInf);
+     h_DY->Add(h_DY5to50_HT100To200); 
+     h_DY->Add(h_DY5to50_HT200To400); 
+     h_DY->Add(h_DY5to50_HT400To600); 
+     h_DY->Add(h_DY5to50_HT600ToInf); 
+     h_DY->Add(h_DY10to50           );
```
Comment (benjaminkreis) [inline plotters/plotter_stacked.C:574]: double counting

File: plotters/plotter_stacked.C [hunk 4]
```diff
@@ -542,69 +567,78 @@ void plotter_stacked()
...
      h_VV->Add(h_ZZToLLLL       ) ;
 
-
-     cout << "h_TTtoLL: " << h_TTtoLL->Integral()<< endl;
-     cout << "h_TTtoLfromTbar: " << h_TTtoLfromTbar->Integral() << endl;
-     cout << "h_TTtoLfromT: " << h_TTtoLfromT->Integral() << endl;
     h_TT = (TH1F*)h_TTtoLL->Clone("h_TT");
      h_TT->Add(h_TTtoLfromTbar );
      h_TT->Add(h_TTtoLfromT    );
+     h_TT->Add(h_TTJets        );
```
Comment (benjaminkreis) [inline plotters/plotter_stacked.C:609]: double counting

File: plotters/plotter_stacked.C [hunk 5]
```diff
@@ -542,69 +567,78 @@ void plotter_stacked()
...
-//     h_Totbkg->Add(h_GJets_HT40To100 ) ;
-//     h_Totbkg->Add(h_GJets_HT100To200) ;
-//     h_Totbkg->Add(h_GJets_HT200To400) ;
-//     h_Totbkg->Add(h_GJets_HT400To600) ;
-//     h_Totbkg->Add(h_GJets_HT600ToInf) ;
+     h_Totbkg->Add(h_DY5to50_HT100To200) ; 
+     h_Totbkg->Add(h_DY5to50_HT200To400) ; 
+     h_Totbkg->Add(h_DY5to50_HT400To600) ; 
+     h_Totbkg->Add(h_DY5to50_HT600ToInf) ; 
+     h_Totbkg->Add(h_DY10to50          ) ;
```
Comment (benjaminkreis) [inline plotters/plotter_stacked.C:625]: double counting

File: plotters/plotter_stacked.C [hunk 6]
```diff
@@ -542,69 +567,78 @@ void plotter_stacked()
...
-     //h_Totbkg->Add(h_STbar_t         ) ;
+     h_Totbkg->Add(h_STbar_t         ) ;
      h_Totbkg->Add(h_ST_t            ) ;
      h_Totbkg->Add(h_STbar_tW        ) ;
      h_Totbkg->Add(h_ST_tW           ) ;
      h_Totbkg->Add(h_TTtoLL          ) ;
+     h_Totbkg->Add(h_TTJets          ) ;
      h_Totbkg->Add(h_TTtoLfromTbar   ) ;
      h_Totbkg->Add(h_TTtoLfromT      ) ;
+     h_Totbkg->Add(h_WW              ) ;
```
Comment (benjaminkreis) [inline plotters/plotter_stacked.C:641]: double counting

File: plotters/plotter_stacked.C [hunk 7]
```diff
@@ -613,7 +647,8 @@ void plotter_stacked()
      h_Totbkg->Add(h_WZTo3LNu        ) ;
      h_Totbkg->Add(h_WZToLNu2QorQQ2L ) ;
      h_Totbkg->Add(h_ZG              ) ;
-     //h_Totbkg->Add(h_ZH_HToBB_ZToLL  ) ;
+     h_Totbkg->Add(h_ZZ              ) ;
```
Comment (benjaminkreis) [inline plotters/plotter_stacked.C:650]: double counting

**Comment by tmrhombus** [comment]
I'd say this is fine to merge in - any objections?

**Comment by benjaminkreis** [comment]
let's do it

**Pull request merged**